src/templates/ingredient.tsv

```diff
@@ -1209,7 +1209,7 @@ CHEBI:31293	Bismuth subnitrate	19477
 CHEBI:31302	Bromazepam	1749
 CHEBI:31305	Bromperidol	19777
 CHEBI:31306	Bronopol	1114345
-CHEBI:31308	brotizolam	19790
+CHEBI:31308	Brotizolam	19790
```

Comment:
Diagnosing this one. ChEBI rarely if ever changes from initial lower to initial upper case letter. In my ChEBI mirror it's lower case, it's lower case on the ChEBI site, etc. So why did it change?

Reply:
Oh I see it got changed back in the subsequent commit. All good here.